.changeset/new-coats-fry.md

```diff
@@ -0,0 +1,5 @@
+---
+"aws-sdk-js-codemod": minor
```

Comment:
Discussed during post-scrum on Tue, June 25th to drop Node.js 14.x support in new major version
```suggestion
"aws-sdk-js-codemod": major
```